docuploader/prepare_java.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import yaml
+from docuploader import log
+
+
+def prepare_java_toc(toc_file, product_name):
```

Comment:
What do you think of pushing this logic to doc-pipeline? If you're developing locally, it would be nice if you also picked up these modifications. But, you shouldn't have to upload and download the YAML to get the updated TOC. If we put this in doc-pipeline, it's slightly more natural, especially combined with Dan's single-command work.

Reply:
Yeah this code doesn't really have a great home. Doc-pipline does make more sense. Closing this PR and moving the code there.